<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputWindow</class>
 <widget class="QWidget" name="InputWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>311</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="NumberInput"/>
    </item>
    <item>
     <widget class="QLineEdit" name="TimeInput"/>
    </item>
    <item>
     <widget class="QLineEdit" name="DestInput"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="NumberLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>24</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер:</string>
   </property>
  </widget>
  <widget class="QLabel" name="TimeLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>79</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Отправление:</string>
   </property>
  </widget>
  <widget class="QLabel" name="DestLabel">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>134</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Станция назначения:</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>190</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="EnterButton">
      <property name="text">
       <string>Ввести</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ClearButton">
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
